miio/gateway.py

```diff
@@ -666,11 +668,6 @@ def get_property(self, property):
                 "Got an exception while fetching property %s" % (property)
             ) from ex
 
-        if response == []:
-            raise GatewayException(
-                "Empty response while fetching property '%s': %s" % (property, response)
-            )
```

Comment:
I think this can stay, but check with `if not response:`. Or am I mistaken, that this command should always be responded with some value (even if it's `''`)?

Reply:
You are right will add it back

Reply:
done